qdev_wrappers/show_num.py

```diff
@@ -4,46 +4,110 @@
 from qcodes.plots.qcmatplotlib import MatPlot
 
 
-def show_num(id, useQT=False, do_plots=True, **kwargs):
+def show_num(id, useQT=False, ave_col=False, ave_row=False, do_plot=True, savepng=False):
     """
     Show  and return plot and data for id in current instrument.
     Args:
         id(number): id of instrument
-        do_plots: Default False: if false no plots are produced.
-        **kwargs: Are passed to plot function
+        useQT (boolean): If true plots with QTplot instead of matplotlib
+        ave_col (boolean): If true subtracts average from each collumn
+        ave_row (boolean): If true subtracts average from each row
+        savepng (boolean): If true saves matplotlib figure as png
 
     Returns:
-        data, plots : returns the plot and the dataset
```

Comment:
Same for this change: maybe it makes it a little more convenient to have this change, but we need to consider if it is worth breaking compatibility here. Let's talk later so that I can learn how everybody is using show_num...

Reply:
again no preference for me. Feel free to change this.